CONTRIBUTING.md

```
@@ -139,6 +139,20 @@ and set the values of the variables according to your environment.
 >     Warn that you should quote the key value with single quotes in the *.env*
 >     file, e.g., `INSTANCE_ACTOR_KEY='{"kty":"RSA",...}'`.
 >
+>  -  `WEB_PUSH_VAPID_PUBLIC_KEY` and `WEB_PUSH_VAPID_PRIVATE_KEY` are used by
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)